sysproxy_test.go

```diff
@@ -0,0 +1,22 @@
+package sysproxy
```

Comment:
Replaces pac_test.go. We no longer test prefix matching since that doesn't apply anymore.